### PR TITLE
share_with

### PR DESCRIPTION
**Pull request by Niteshkr123**
Added a feature to share with various platforms. User will now get an option to share once clicked on more_vert icon

### CODE REVIEW

**Comment by anushbhatia** [comment]
add this in the app drawer instead of the navbar

**Comment by Niteshkr123** [comment]
okay, that means the user can see an option of share whenever he will click on the drawer. Right?

**Comment by anushbhatia** [comment]
yep

**Comment by Niteshkr123** [comment]
Hello sir, 
I think you forgotten to add tags in this PR. I request you to please tag in it.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
```
Comment (smaranjitghose) [inline .vscode/launch.json:6]: .gitignore these

Comment (Niteshkr123) [reply .vscode/launch.json:6]: I have fixed this and have made a new PR. Kindly review that.

**Pull request merged**